nltk/collocations.py

```diff
@@ -66,18 +67,18 @@ def _build_new_documents(cls,documents, window_size, pad_left=False, pad_right=F
             return _itertools.chain.from_iterable(_itertools.chain(doc, padding) for doc in documents)
         if pad_left:
             return _itertools.chain.from_iterable(_itertools.chain(padding, doc) for doc in documents)
-    
+
     @classmethod
     def from_documents(cls, documents):
         """Constructs a collocation finder given a collection of documents,
         each of which is a list (or iterable) of tokens.
         """
-        #return cls.from_words(_itertools.chain(*documents))
-        return cls.from_words(cls._build_new_documents(documents, cls.default_ws, pad_right=True)) 
+        # return cls.from_words(_itertools.chain(*documents))
```

Comment:
There is a convention: for commented out lines there is no space between `#` and code; it allows to visually distinguish them from comments.